test/backend_test.in.cpp

```diff
@@ -483,6 +483,35 @@ TEST(${BACKEND_NAME}, concat_vector)
     EXPECT_EQ((vector<float>{2, 4, 8, 16, 1, 2, 4, 8, 16, 32, 18, 19}), read_vector<float>(result));
 }
 
+TEST(${BACKEND_NAME}, concat_mkldnn)
```

Comment:
Why do we need a special test case for mkldnn?

Reply:
The other concat test cases did not have 4D inputs and i only enable concat for mkldnn for 4D inputs. Since you are specifying the API should be able to handle any dimension inputs, i guess we dont need this anymore.

Reply:
you can add it as concat_4d_tensor instead of calling it concat_mkldnn